### DISCUSSION TITLE
q-dialog problem with CKEditor5

### DISCUSSION THREAD

**Discussion by kimchulyeon** [quasarframework/quasar#15037] [General - Components / Directives / etc]
Hello. I'm using CkEditor5 and face a problem with q-dialog. I think it's an auto focus issue.
When I use CKEditor5 inside the q-dialog. table tooltip bar and image tooltip bar are not shown. So I fixed with CSS... but when I click the table tooltip bar buttons, it happens nothing and tooltip bar turned off. I thought it's auto focus issue so I gave "no-focus" and "no-refocus" options to q-dialog. But it doesn't solve my problem.... Can I get some advice 🥲

**Comment by metalsadman** [comment]
try using `allow-focus-outside` on your QDialog.

**Reply by kimchulyeon** [reply]
I tried...but it's not working. Sir, then is it not a focus issue?

**Reply by kimchulyeon** [reply]
q-dialog seamless option helps but I don't want users to scroll or click outside of the modal..🥲

**Reply by metalsadman** [reply]
I don't know then, not inclined to roll my own reproduction using a lib i don't know about. Best put it out in a stackblitz so we can check.

**Reply by kimchulyeon** [reply]
really appreciate it! I just choose the easy way. I hope I can find any info in GOOGLE.
